addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ZestZapRunner.java

```diff
@@ -388,13 +392,53 @@ public void handleResponse(ZestRequest request, ZestResponse response)
     public String handleClient(ZestScript script, ZestClient client)
             throws ZestClientFailException {
         try {
+            if (client instanceof ZestClientLaunch) {
+                String res = this.launchClient((ZestClientLaunch) client);
+                if (res != null) {
+                    return res;
+                }
+            }
             return super.handleClient(script, client);
         } catch (Exception e) {
             LOGGER.error(e.getMessage(), e);
             return null;
         }
     }
 
+    private String launchClient(ZestClientLaunch clientLaunch) {
+        ExtensionSelenium extSel =
+                Control.getSingleton().getExtensionLoader().getExtension(ExtensionSelenium.class);
+        Browser browser = null;
+
+        if ("Firefox".equalsIgnoreCase(clientLaunch.getBrowserType())) {
```

Comment:
Instead of using literals why not use `Browser.FIREFOX.getId()`?
(Same for chrome)

Reply:
Unlikely that we'll change the IDs but they are unrelated to Zest ones.